components/kyma-environment-broker/internal/provisioner/client.go

```diff
@@ -132,9 +155,27 @@ func (c *client) RuntimeOperationStatus(accountID, operationID string) (schema.O
 	req.Header.Add(accountIDKey, accountID)
 
 	var response schema.OperationStatus
-	err := c.graphQLClient.ExecuteRequest(req, &response)
+	err := c.executeRequest(req, &response)
 	if err != nil {
 		return schema.OperationStatus{}, errors.Wrap(err, "Failed to get Runtime operation status")
 	}
 	return response, nil
 }
+
+func (c *client) executeRequest(req *gcli.Request, respDestination interface{}) error {
+	if reflect.ValueOf(respDestination).Kind() != reflect.Ptr {
+		return errors.New("destination is not of pointer type")
+	}
+
+	type graphQLResponseWrapper struct {
+		Result interface{} `json:"result"`
+	}
+
+	wrapper := &graphQLResponseWrapper{Result: respDestination}
+	err := c.graphQLClient.Run(context.Background(), req, wrapper)
```

Comment:
here we should have `context.TODO()` as this request should be canceled on shutdown